### PR TITLE
Import Github API to trigger workflow and fetch issues

### PR DESCRIPTION
**Pull request by tronghieuvuong**
### Summary
Use Github API to trigger workflow in this repo https://github.com/bcgov/metrics-report/ pull the metrics for Entity team and display its output issue to frontend. Sample issue that is expected to fetch: https://github.com/bcgov/metrics-report/issues/65

Frontend: Display the raw response fetched from the issue

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-41-ojo7tfb1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-41-v7ku8awj.web.app

File: apps/application-status/composables/getIssues.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { getOctokit } from '~/githubClient'
+
+async function getIssue (owner: string, repo: string, teamName: string) {
+  const issues = await getOctokit().request('GET /repos/{owner}/{repo}/issues', {
+    owner,
+    repo,
+    labels: teamName,
+    headers: {
+      'X-GitHub-Api-Version': '2022-11-28'
```
Comment (bolyachevets) [inline apps/application-status/composables/getIssues.ts]: maybe parameterize the date 2022-11-28, as it is repeated in multiple files

File: apps/application-status/package-lock.json [hunk 2]
```diff
@@ -1,15 +1,18 @@
 {
   "name": "application-status",
+  "version": "1.0.0",
```
Comment (bolyachevets) [inline apps/application-status/package-lock.json:3]: need to start incrementing this version

File: apps/application-status/.env.example [hunk 3]
```diff
@@ -1 +1,2 @@
-ZENHUB_APIKEY
\ No newline at end of file
+ZENHUB_APIKEY
+GITHUB_TOKEN
```
Comment (bolyachevets) [inline apps/application-status/.env.example:2]: where are you pulling GITHUB_TOKEN from? If you want this variable to be in gcbrun, you will need to add it here: https://github.com/bcgov/sbc-producthub/blob/069ac47b532e8cc61505f57e395ad06b3ec051ad/apps/application-status/devops/cloudbuild-pr.yaml#L10
If you also want it to be loaded in the builds from 1password, you will need to add it here https://github.com/bcgov/sbc-producthub/blob/main/apps/application-status/vaults.env probably something like GITHUB_TOKEN ="op://API/$APP_ENV/github/GITHUB_TOKEN"

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-41-ojo7tfb1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-41-v7ku8awj.web.app

File: apps/application-status/.env.example (hunk 3 above)
Comment (tronghieuvuong) [reply apps/application-status/.env.example:2]: I think I did not have the permission to add the key on GCP or 1password yet, therefore, gcbrun build is currently failing to display, and the same for dev build

Comment (bolyachevets) [reply apps/application-status/.env.example:2]: I sent you direct message about the key value, as I don't know which value you are using

File: apps/application-status/vaults.env [hunk 4]
```diff
@@ -1,2 +1,3 @@
 #vaults api
-ZENHUB_APIKEY="op://API/$APP_ENV/zenhub/ZENHUB_APIKEY"
\ No newline at end of file
+ZENHUB_APIKEY="op://API/$APP_ENV/zenhub/ZENHUB_APIKEY"
+GITHUB_TOKEN="op://API/$APP_ENV/zenhub/ZENHUB_APIKEY"
```
Comment (bolyachevets) [inline apps/application-status/vaults.env]: this should be GITHUB_TOKEN="op://API/$APP_ENV/github/GITHUB_TOKEN""

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-41-ojo7tfb1.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-41-v7ku8awj.web.app

**Review by bolyachevets — APPROVED** [review]

**Pull request merged**